src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-configurator/filter-prometheus-metrics.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Filter Prometheus Metrics
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus Configurator
+metaDescription: Avoid exceeding New Relic's platform limits and increasing your billing charges by using the Prometheus Configurator's filtering capabilities.
+---
```

Comment:
It seems like this page heavily needs users to have read `set-up-prometheus-agent-configurator.mdx`. makes me wonder if we should merge both in a single page to avoid confusion 🤔

Reply:
Merged.